katana_supported_configs.txt

```diff
@@ -0,0 +1,5 @@
+linux_64_cuda_compiler_versionNonecudnnundefinednumpy1.18python3.8.____cpython
+linux_64_cuda_compiler_version10.2cudnn7numpy1.18python3.8.____cpython
+linux_64_cuda_compiler_version11.0cudnn8numpy1.18python3.8.____cpython
+linux_64_cuda_compiler_version11.1cudnn8numpy1.18python3.8.____cpython
+linux_64_cuda_compiler_version11.2cudnn8numpy1.18python3.8.____cpython
```

Comment:
This could potentially go out of sync with up stream. That's OK. I don't have a good way around it. But it's worth being aware of. Maybe make a note in the README or something.